test/AsmJs/rlexe.xml

```diff
@@ -978,7 +978,7 @@
       <compile-flags>-testtrace:asmjs -args 14000 -endargs -EnableFatalErrorOnOOM-</compile-flags>
       <!-- todo:: On unix platforms there is more stack available,
            so we need to find the right limit to test in order to not timeout -->
-      <tags>exclude_dynapogo</tags>
+      <tags>exclude_dynapogo,exclude_mac</tags>
```

Comment:
The TODO above doesn't line up with the PR description -- should it be removed or changed to clarify why we are now excluding mac? Also, I don't know exactly how the `Slow` tag is used, but I know it exists for other tests, if thats preferable.

Reply:
It's not exactly clear, but I think the idea is that we want a count that is high enough to test out of stack but low enough to not time out. Seems on osx we hit timeout first. It's not really that important to test on all platforms, so I'm gong to leave it as is.